cloudformation/aws-parallelcluster.cfn.json

```diff
@@ -317,8 +322,8 @@
     "RAIDOptions": {
       "Description": "Comma Separated List of RAID related options, 8 parameters in total, [shared_dir,raid_type,num_of_raid_volumes,volume_type,volume_size,volume_iops,encrypted,ebs_kms_key_id]",
       "Type": "String",
-      "Default": "NONE,NONE,NONE,NONE,NONE,NONE,NONE,NONE",
-      "AllowedPattern": "^(NONE|.+)(,|, )(NONE|\\d)(,|, )(NONE|\\d)(,|, )(NONE|standard|io1|io2|gp2|st1|sc1)(,|, )(NONE|\\d+)(,|, )(NONE|\\d+)(,|, )(NONE|true|false)(,|, )(NONE|.+)$"
+      "Default": "NONE,NONE,NONE,NONE,NONE,NONE,NONE,NONE,NONE",
+      "AllowedPattern": "^(NONE|.+)(,|, )(NONE|\\d)(,|, )(NONE|\\d)(,|, )(NONE|standard|io1|io2|gp2|gp3|st1|sc1)(,|, )(NONE|\\d+)(,|, )(NONE|\\d+)(,|, )(NONE|true|false)(,|, )(NONE|.+)$"
```

Comment:
If I am understanding correctly we are chaining 9 parameters together now?
If so, we need to update `AllowedPattern ` to check for 9 parameters, it only check for 8 now.

Reply:
Thanks! Fixed